api/apps/v1alpha1/nemo_datastore_types.go

```diff
@@ -592,10 +593,7 @@ func (n *NemoDatastore) GetDefaultLivenessProbe() *corev1.Probe {
 		ProbeHandler: corev1.ProbeHandler{
 			HTTPGet: &corev1.HTTPGetAction{
 				Path: "/v1/health/live",
-				Port: intstr.IntOrString{
-					Type:   intstr.Type(0),
-					IntVal: n.Spec.Expose.Service.Port,
-				},
+				Port: getProbePort(n.Spec.Expose.Service),
```

Comment:
This implies that `.spec.expose.service` is required, but it is not currently.
Maybe we should make it a required field or add a default value?

Reply:
yes, we can make this as a required field, earlier there was just single port param and default for that, so it was always present.